src/Resolver/PathResolver.php

```diff
@@ -77,6 +77,6 @@ public function createPath(string $relativePath = ''): string
      */
     public function getProjectDir(): string
     {
-        return $this->projectDirEnv ?? (string)getcwd();
+        return $this->projectDirEnv ?: (string)getcwd();
```

Comment:
Fair point, but in this case, typehint of the property also should be changed, isn't it?